learningresources/tests/base.py

```diff
@@ -155,5 +177,5 @@ def get_course_single_tarball(self):
         Returns:
             path (unicode): absolute path to tarball.
         """
-        path = join(abspath(dirname(__file__)), "testdata", "courses")
-        return self.copy_file(join(path, "single.tgz"))
+        path = join(abspath(dirname(__file__)), "testdata", "courses", "toy")
+        return self._make_archive(path, False, '.tgz')
```

Comment:
so the make_archive() doesn't actually have anything to do with the file name that is under test.  It is just done in a temp directory which is saved to the filesystem for use by all code as the filename defined at https://github.com/mitodl/lore/pull/222/files#diff-589006e737018a1af16a1e95dbf5f35bR137.  So the ext parameter (with defaults to .zip and .tar.gz respectively) is what determines the extension under test, and exercises our extension verifying code in the import.  (From the standup, and these comments, I wanted to double check that was true which is why I asked you to hold off on the merge).